ui/components/atoms/A007 - Custom Buttons with CSS/index.js

```diff
@@ -0,0 +1,80 @@
+// List Libraries being used
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+import { withStyles } from '@material-ui/core/styles'
+
+// Import from UI library
+import Button from '@material-ui/core/Button'
+
+// Import JSON schema
+import schema from './schema'
+
+// Import styles here - see example setup for styles at ui/example/components/atoms/checkbox
+
+// custom styling
+
+// 1. CSS Stylesheet
+import './custom.css';
```

Comment:
remove .css import - we are using style.js

Reply:
I removed this whole A007